python/cudf/dataframe/index.py

```diff
@@ -18,6 +18,7 @@
 
 
 class Index(object):
+    is_monotonic = None
```

Comment:
Since Pandas expects a `True` or `False` here will this cause issues being `None` for now?

Reply:
Pandas won't actually look at this code.  Currently there are two consumers:

1.  dask.dataframe, which is looking for truthiness to see if it should use smarter `loc` algorithms (which cudf won't support)
2.  an eager user, who will get a possibly confusing result.

So it's slightly sinful to leave this as None, but it seems better than stating False (when it might not be true).  This is generally a bit gross, but I'm inclined to push on.

Reply:
For Dask dataframe repartition algorithms it would be nice to Be able to access `df.index.is_monotonic` and have it not err.